thin-replica-server/proto/state_snapshot_concord.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 VMware, all rights reserved
+//
+// Concord Client's StateSnapshot Service
+
+syntax = "proto3";
+package com.vmware.concord.state_snapshot_concord;
+
+option java_package = "com.vmware.concord.state_snapshot_concord";
+
+// The Concord StateSnapshot service can be used by the concord client to 
+// retrieve state snapshot from a concord cluster
+// See `client/proto/state_snapshot.proto` for definition of state snapshot.
```

Comment:
The path is not correct. Instead, I suggest `client/proto/state_snapshot` to point to the directory instead of a specific version of the interface.